iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/KubernetesSensor.java

```diff
@@ -128,15 +135,24 @@ protected FilePredicate customFilePredicate(SensorContext sensorContext) {
     var helmTemplatePredicate = predicates.and(
       predicates.matchesPathPattern("**/templates/**"),
       new HelmProjectMemberPredicate(sensorContext));
-    var valuesYamlPredicate = predicates.and(
+    var valuesYamlOrChartYamlPredicate = predicates.and(
       predicates.or(
         predicates.matchesPathPattern("**/values.yaml"),
```

Comment:
If I understand the docs correctly, `or()` and multiple `matchesPathPattern` is the same as one `matchesPathPatterns(array)`